### DISCUSSION TITLE
CI/CD pipeline problem

### DISCUSSION THREAD

**Discussion by TaoZex** [apache/doris#12748] [A - General / Q&A]
I submitted some documents to the Doris community.In P1 Regression, the details of the error message are as follows:
<img width="862" alt="details" src="https://user-images.githubusercontent.com/45089228/191038698-26bf1053-09bc-4ac4-b020-e0c3a4b6b358.png">
I submitted code to the doris community once before, and the CI pipeline also encountered an error that the Mysql server could not be connected. After multiple submissions, it finally passed the pipeline.May I ask why such a problem occurs?

**Comment by zy-kkk** [comment]


**Reply by TaoZex** [reply]
May I ask if updating my warehouse will trigger the check again? For example:
<img width="852" alt="0920question" src="https://user-images.githubusercontent.com/45089228/191155768-b8be5d60-2848-4279-a078-0affd9c02998.png">

**Reply by zy-kkk** [reply]
Yes, updating the branch in this way retriggers the check

**Reply by TaoZex** [reply]
I learned a lot from communicating with you. I have marked the answer as the answer.Thank you!
